### ISSUE TITLE
Cannot download ZooeyBot due to "infected file."

### ISSUE THREAD

**Issue by Rocxidi** [Masuzu/ZooeyBot#1316]
When trying to download the file from Google Drive, an error message shows up saying "Sorry, this file is infected with a virus

Only the owner is allowed to download infected files."

The other download link doesn't work.

**Comment by huehueno** [comment]
Yeah the Google Drive links won't work since the folder is packed with the key generator in it. Can't access the files I've paid for yet. =/
